user/abstractions/operations.md

```diff
@@ -29,7 +29,9 @@ Check if a user has **Operations** authority.
 
 #### ❌ Raises
 * `SecurityRequestError`<br>
-  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is equal to `4`.
+* `DownstreamFatalError`<br>
+  Raises `DownstreamFatalError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is greater than `4`.
 
 #### 💻 Example
 

```

Comment:
The exceptions for `take_away_operations_authority()` still needs to be updated.

Reply:
Not sure how `operations.md` got missed in my initial crawl but there were two places that needed a touch up. Both addressed.